urls.py

```diff
@@ -6,11 +6,11 @@
     'corehq.apps.prelogin.views',
     url(r'^home/$', HomePublicView.as_view(), name=HomePublicView.urlname),
     url(r'^impact/$', ImpactPublicView.as_view(), name=ImpactPublicView.urlname),
-    url(r'^software_services/$', SoftwareServicesPublicView.as_view(),
+    url(r'^pricing/$', SoftwareServicesPublicView.as_view(),
         name=SoftwareServicesPublicView.urlname),
     url(r'^services/$', ServicesPublicView.as_view(),
         name=ServicesPublicView.urlname),
-    url(r'^pricing/$', PricingPublicView.as_view(),
+    url(r'^services/$', PricingPublicView.as_view(),
```

Comment:
@calellowitz this is where [FB 220372](http://manage.dimagi.com/default.asp?220372) happened -- there are two URLs with the same pattern. ... But the other issue is the `LANGUAGE_CODE` in [build.html][2]. `LANGUAGE_CODE` is not given as an arg in the base template, so I'm guessing it's safe to drop. If you choose to close this PR, you could just cherry pick 8c127cd.

  [2]: https://github.com/dimagi/commcarehq-prelogin/commit/c2d9d03d27cbec030a78bc30d2f12148a8c6de76#diff-2050d70e1023883852b1f6d637ff8b36R93